presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveArrayFunctionQueries.java

```diff
@@ -45,4 +45,21 @@ public void testShuffle()
         this.assertQuerySucceeds("SELECT shuffle(array_sort(quantities)) FROM orders_ex");
         this.assertQuery("SELECT array_sort(shuffle(quantities)) FROM orders_ex");
     }
+
+    @Test
+    public void testArrayMatch()
+    {
+        final String errorMessage = ".*division by zero presto.default.divide.*";
+        this.assertQuerySucceeds("SELECT all_match(quantities, x -> ((10 / x) > 2)) FROM orders_ex");
```

Comment:
Why not assertQuery? It would be nice to ensure not only query doesn't fail, but also returns correct results.

Reply:
Fixed